modules/nw-sriov-configuring-operator.adoc

```diff
@@ -96,6 +96,10 @@ By default, this field is set to `2`.
 |`boolean`
 |Specifies whether to enable or disable the SR-IOV Network Operator metrics. By default, this field is set to `false`.
 
+|`spec.featureGates.mellanoxFirmwareReset`
+|`boolean`
+|Specifies whether to reset the firmware in the SR-IOV Network Operator. By default, this field is set to `true`.
```

Comment:
Feel free to re-phrase my suggestion:

```suggestion
|Specifies whether to reset the firmware on VF changes in the SR-IOV Network Operator. Certain chipsets (e.g., Intel C740 Series) do not completely power off the PCI-E devices, which is required to configure VFs on NVIDIA/Mellanox NICs. By default, this field is set to `false`.
```

Reply:
Thanks @cgoncalves for your feedback. I have included, and slightly re-phrased, your suggestion.